docs/source/conf.py

```diff
@@ -12,25 +12,26 @@
 #
 import os
 import subprocess
+
 # sys.path.insert(0, os.path.abspath('.'))
 
 current_file_dir = os.path.dirname(os.path.realpath(__file__))
 
+
 def get_git_short_hash():
-    rc = subprocess.check_output(['git', 'rev-parse', '--short', 'HEAD'])
+    rc = subprocess.check_output(["git", "rev-parse", "--short", "HEAD"])
```

Comment:
is using double quotes the recommended way?

Reply:
The python code formater used here is Black. As per black code style double quotes in preferred (https://black.readthedocs.io/en/stable/the_black_code_style.html#strings).